zerver/lib/timestamp.py

```diff
@@ -11,3 +11,15 @@ def timestamp_to_datetime(timestamp):
 def datetime_to_timestamp(datetime_object):
     # type: (datetime.datetime) -> int
     return calendar.timegm(datetime_object.timetuple())
+
+def utc_datetime_to_psql_timestamptz(datetime_object):
```

Comment:
Are we using this? Also we should store timezone aware objects in the database if possible (let me know if that's unfamiliar).